app/routes.py

```diff
@@ -0,0 +1,413 @@
+from app import db
+from app.models.customer import Customer 
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, request
+import datetime
+from datetime import timedelta 
+
+
+# Establishing due date
+NOW = datetime.datetime.now()
+DUE_DATE = NOW + timedelta(days=7)
+
+# Blueprints
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+# Customer model helper functions
+def display_customer_info(customer):
+    return {
+        "id": customer.id,
+        "name": customer.name,
+        "postal_code": customer.postal_code,
+        "phone": customer.phone
+    }
```

Comment:
Nice helper function. We could move this function into the `Customer` model and that way every instance of a `Customer` will have access to this method, eliminating the need to pass in the instance as a parameter.